File: prettyprint3.py
def seqprinter(num):
    string=""
    for i in range(1,num+1):
        string+=str(num*i)+" "
    print string

def mainprinter(num):
    for i in range(1,num+1):
        seqprinter(i)


# seqprinter(5)
if __name__ == '__main__':
    x = int(raw_input("Enter N: "))
    mainprinter(x)

File: prettyprint3.c
#include <stdio.h>



int main()

{

    int i, j, N;



    printf("Enter N: ");

    scanf("%d", &N);



    for(i=1; i<=N; i++)

    {

        for(j=i; j<=(i*i); j += i)

        {

            printf("%-3d", j);

        }



        printf("\n");

    }



    return 0;

}
